packages/suite-web/pages/_document.tsx

```diff
@@ -5,6 +5,11 @@ import React from 'react';
 import { AppRegistry } from 'react-native';
 import globalStyles from '@suite-support/styles/global';
 
+// This component is used to render the static inline HTML
+// which is used in ./static/js/browserDetection.js to render
+// it in unsupported browsers
+// import UnsupportedBrowser from './unsupported';
+
```

Comment:
remove this comment and whole UnsupportedBrowser component since it's not used

Reply:
It was meant to be used in case we want to change the content/layout/design of the unsupported browser page. 
But since you can edit that directly in your browser it's unnecessary.
Removed